EugeneMunya/QuickCredit3
# #166183355 Change loan status to reject or approved in the database.

#### What does this PR do?
Add functionality that allows an admin to update loan status. 
#### Description of Task to be completed?
At this point, an admin will be able to change loan status to an approved or rejected.
#### How should this be manually tested?
After cloning this repo and running npm start, then open postman and use a link looks like this 
localhost:3000/api/v1/loans/5
the last number is a loan id, but make sure that you have logged in.
and pass your token as an authorization header and status value as you the image below
#### What are the relevant pivotal tracker stories?
#166183355
#### Screenshots
![db](https://user-images.githubusercontent.com/36619897/58251363-bd240700-7d63-11e9-986d-d08585d46d1a.jpeg)

#### Questions
No questions
